srcts/src/components/tooltip.ts

```diff
@@ -77,25 +81,27 @@ export class BslibTooltip extends LightElement {
   connectedCallback(): void {
     super.connectedCallback();
 
-    const el = this.triggerElement;
-    el.setAttribute("data-bs-toggle", "tooltip");
-    el.setAttribute("tabindex", "0");
-    this.tooltip = new bsTooltip(el, this.options);
+    // The user-supplied content is wrapped up in to an additional <div> (this
+    // guarantees that we can pass an _Element_ to bootstrap.Tooltip(), which
+    // moves the content from within this component to the tooltip's location).
+    // These inline styles are here to prevent any styling suprises caused by
+    // the wrapping <div>.
+    this.content.style.display = "contents";
```

Comment:
Could we set this in the R constructor instead? Also I'm curious why the extra div is required (could we move the parent div)?

Reply:
I gave this some more thought and opt-ed to keep it the way it is. I like that `display:contents;` is not on the R side since it does require a mental model of the web component to understand why it's there. I also had a look at getting rid of the extra `div(style="display:none")`, but it didn't seem like an obvious win in terms of simpler logic